orders/OrderFormNotion.ts

```diff
@@ -68,4 +78,14 @@ export default class OrderForm {
             console.warn(`Order form checker: An error occurred when checking for order form updates.`, e);
         }
     }
+
+    private completedSushiOrder(orderObject: PageObjectResponse) {
+        if (orderObject.properties["Order Description"]?.type == "title" && orderObject.properties["Product Name"]?.type == "rich_text" && orderObject.properties["Subtotal"]?.type == "number") {
```

Comment:
minor: can we extract these props to consts? Similar to what we have in `NotionClient.ts`

Reply:
And maybe we can use those consts in `OrderBot.ts` too

Reply:
I think there's some larger refactoring I can do! I'm going to do this and some more in a diff PR